src/main/webapp/app/app.component.ts

```diff
@@ -1,5 +1,9 @@
-import {Component, OnInit} from '@angular/core';
```

Comment:
You forgot to adapt the unit test

Reply:
Added all unit tests except for the task service. We should wait until the HTTP parts are implemented.